docs/en/operations/settings/settings.md

```diff
@@ -1262,4 +1262,23 @@ Possible values:
 
 Default value: 16.
 
+## always_fetch_merged_part {#always_fetch_merged_part}
+
+Prohibits data parts merging at replicas in [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md)-engine tables.
+
+If enabled, replica never merges parts and always downloads merged parts from other replicas. If there is no merged data yet, the replica waits for them to appear. 
```

Comment:
Merge this and the following paragraph into one. Consider starting the united paragraph with the "When merging is prohibited..." phrase instead of "If enabled".

`... **the** replica never ..."